assets/test/spec/controllers/project.js

```diff
@@ -16,18 +17,18 @@ describe('Controller: ProjectController', function () {
       this.page = function() {return this;};
       this.subPath = function() {return this;};
       this.build = function() {return this;};
-      this.join = function() {return "";};
+      this.join = function() {return '';};
       this.$get = function() {return new Mocked();};
     });
 
-    $provide.factory("HawtioNav", function(){
+    $provide.factory('HawtioNav', function(){
       return {add: function() {}};
     });
   }));
 
   // Make sure a base location exists in the generated test html
   if (!$('head base').length) {
-    $('head').append($('<base href="/">'))
+    $('head').append($('<base href='/'>'));
```

Comment:
is the single-quote thing enforced by jshint? I thought they were trying to limit themselves to correctness... I don't mind it, I just wouldn't change a lot of existing strings